homeassistant/components/nobo_hub/translations/no.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
Please do not include additional languages (only the main `strings.json` and `en.json`)

Reply:
Deleted. And I've signed up for Lokalise. :)